docs/development_environment.md

```diff
@@ -30,35 +30,50 @@ Due to Home Assistant is mainly designed and developed on Linux distributions, o
 
 Open Powershell as an Administrator and run
 
-```bash
+```powershell
 Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
 ```
 
 From Windows Store install Ubuntu.
 
 When the Linux subsystem is set up, perform install as for Linux.
 
+Install the core dependencies.
+
+```bash
+sudo apt-get update
+sudo apt-get install python3-pip python3-dev python3-venv
+```
+
+> It might be a good idea to upgrade to Python 3.6 in Linux Subsystem on Windows. Instructions for Ubuntu 16.04:
```

Comment:
There is no reason to make development instructions more complicated, as we always support the Python in Debian Stable, please revert remove this.

Reply:
But you did not when I installed?!
I got an error that 3.5.2 (if I remember correctly) was not supported and I should update but there was no official update for Debian at the time. Why not make a quicker dev-repository for a faster release cycle in dev-machines?

Reply:
Debian ships Python 3.5.3.

Reply:
@fabaff I have Ubuntu installed as a Subsystem on my Windows-machine (it installs 16.04 version of Ubuntu).
I have run all updates with apt-get and updated all packages. This still installs Python 3.5.2 (that is not supported).
Source: https://packages.ubuntu.com/search?keywords=python3.5

![image](https://user-images.githubusercontent.com/10034/52875726-5637f600-3155-11e9-9e40-a10736411f56.png)

Reply:
@fabaff Ubuntu for some reason choose to not include 3.5.3(Only 3.5.2) in Ubuntu 16.04 which is based on Debian Stretch. Since that's the version that's shipped with the Subsystem it would be preferable to have instructions to a workaround.